shipstation/api.py

```diff
@@ -398,3 +418,23 @@ def post(self, endpoint='', data=None):
         )
         if self.debug:
             pprint.PrettyPrinter(indent=4).pprint(r.json())
+
+    def fetch_orders(self, parameters_dict=None):
+        invalid_parameters = []
+
+        if parameters_dict is None:
```

Comment:
Given we're demanding a dict here, I'd prefer that the default arg be `{}` instead of `None`, and a guard clause be added to raise an exception if something else is passed.

```python
def fetch_orders(self, parameters_dict={}):
    if not isinstance(parameters_dict, dict):
        raise AttributeError("parameters_dict must be of type dict")
```